### PR TITLE
refactor: Normalize the usage of the `useSetting` hook

### PR DESCRIPTION
**Pull request by tassoevan**
## Proposed changes (including videos or screenshots)

It reviews how some calls of the `useSetting()` hook are performed. It replaces type coercions (e.g. `String(useSetting('xyz'))` and type assertions (e.g. `useSetting('xyz') as string`) with fallback values (e.g. `useSetting('xyz', 'fallback')`) and explicit generics (e.g. `useSetting<string>('xyz')`).

### CODE REVIEW

File: apps/meteor/client/views/room/composer/messageBox/MessageBoxActionsToolbar/hooks/useVideoMessageAction.ts [hunk 1]
```diff
@@ -9,10 +9,10 @@ import { useMediaActionTitle } from '../../hooks/useMediaActionTitle';
 import { useMediaPermissions } from '../../hooks/useMediaPermissions';
 
 export const useVideoMessageAction = (disabled: boolean): GenericMenuItemProps => {
-	const isFileUploadEnabled = useSetting('FileUpload_Enabled') as boolean;
-	const isVideoRecorderEnabled = useSetting('Message_VideoRecorderEnabled') as boolean;
-	const fileUploadMediaTypeBlackList = useSetting('FileUpload_MediaTypeBlackList') as string;
-	const fileUploadMediaTypeWhiteList = useSetting('FileUpload_MediaTypeWhiteList') as string;
+	const isFileUploadEnabled = useSetting('FileUpload_Enabled', true);
+	const isVideoRecorderEnabled = useSetting('Message_VideoRecorderEnabled', true);
+	const fileUploadMediaTypeBlackList = useSetting('FileUpload_MediaTypeBlackList', '');
```
Comment (aleksandernsilva) [inline apps/meteor/client/views/room/composer/messageBox/MessageBoxActionsToolbar/hooks/useVideoMessageAction.ts:14]: The default value for this setting is `'image/svg+xml'`. Should it be reflected here?

File: apps/meteor/client/views/room/composer/messageBox/MessageBoxActionsToolbar/hooks/useAudioMessageAction.ts [hunk 2]
```diff
@@ -11,10 +11,10 @@ import { useMediaPermissions } from '../../hooks/useMediaPermissions';
 const audioRecorder = new AudioRecorder();
 
 export const useAudioMessageAction = (disabled: boolean, isMicrophoneDenied: boolean): GenericMenuItemProps => {
-	const isFileUploadEnabled = useSetting('FileUpload_Enabled') as boolean;
-	const isAudioRecorderEnabled = useSetting('Message_AudioRecorderEnabled') as boolean;
-	const fileUploadMediaTypeBlackList = useSetting('FileUpload_MediaTypeBlackList') as string;
-	const fileUploadMediaTypeWhiteList = useSetting('FileUpload_MediaTypeWhiteList') as string;
+	const isFileUploadEnabled = useSetting('FileUpload_Enabled', true);
+	const isAudioRecorderEnabled = useSetting('Message_AudioRecorderEnabled', false);
```
Comment (aleksandernsilva) [inline apps/meteor/client/views/room/composer/messageBox/MessageBoxActionsToolbar/hooks/useAudioMessageAction.ts:15]: The default value for this setting is `true`. Should it be reflected here?

**Review by aleksandernsilva — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/room/composer/messageBox/MessageBoxActionsToolbar/hooks/useShareLocationAction.tsx
Comment (dougfabris) [inline apps/meteor/client/views/room/composer/messageBox/MessageBoxActionsToolbar/hooks/useShareLocationAction.tsx:1]: no fallback for this?

File: apps/meteor/client/views/room/body/RoomBodyV2.tsx
Comment (dougfabris) [inline apps/meteor/client/views/room/body/RoomBodyV2.tsx:1]: fallback?

File: apps/meteor/client/views/omnichannel/installation/Installation.tsx
Comment (dougfabris) [inline apps/meteor/client/views/omnichannel/installation/Installation.tsx:1]: fallback?

File: apps/meteor/client/hooks/useTimeAgo.ts
Comment (dougfabris) [inline apps/meteor/client/hooks/useTimeAgo.ts:1]: fallback?

**Review by dougfabris — COMMENTED** [review]
Overall looks greats, just thinking about some missing fallbacks

**Review by dougfabris — APPROVED** [review]
LGTM!

**Review by aleksandernsilva — APPROVED** [review]

**Review by ggazzo — APPROVED** [review]

**Pull request merged**